src/hooks/cgroups/pbs_cgroups.PY

```diff
@@ -2388,8 +2388,8 @@ class CgroupUtils(object):
                 prefix = ''
             else:
                 prefix = subsys + '.'
-        if self.systemd_version < 205:
-            return os.path.join(mnt_point, self.cfg['cgroup_prefix'], prefix)
+#        if self.systemd_version < 205:
+#            now return the exact same path -- do not try to elide ".slice"
```

Comment:
If we no longer need the code, please remove the lines rather than comment them. I'm not convinced that is the case here. I think this change should be reverted.